README.md

```diff
@@ -42,6 +38,17 @@ Documentation | Package
 
 Read the [contributing guidelines](CONTRIBUTING.md) file if you are interested in contributing.
 
+## Getting Started
+
+  1. Install [Node.js and npm](https://nodejs.org/en) <!-- TODO: use a dev container or NixOS maybe? -->
+      - Use the correct [engine version](package.json): [n](https://www.npmjs.com/package/n), [nvm](https://github.com/nvm-sh/nvm), or other
+  2. Use `pnpm` by default `corepack enable pnpm`
+  3. Install dependencies project `npm install`
+  4. Check that gravity is still working - `npm test` - before making any changes
+  5. Add your ssh key to your account
```

Comment:
We might want to move this portion to the `CONTRIBUTING.md` file. Also not that we may need to word things to accommodate both internal contribution (e.g. ssh key flow) as well as external e.g. forking flow.

Reply:
That might be a better spot yeah, although small preference towards `documentation/` as the folder name